parity-util-mem/Cargo.toml

```diff
@@ -17,10 +17,10 @@ elastic-array = { version = "*", optional = true }
 ethereum-types = { version = "*", optional = true }
 parking_lot = { version = "*", optional = true }
 
-[target."cfg(windows)".dependencies.winapi]
+[target.'cfg(target_os = "windows")'.dependencies.winapi]
```

Comment:
cfg(windows) should actually work according to [reference](https://doc.rust-lang.org/reference/conditional-compilation.html#set-configuration-options) (equivalent to `target_family = "windows"`), but I prefer `target_os` for consistency with other OSs :+1: